src/main/java/com/iota/iri/service/snapshot/impl/SnapshotProviderImpl.java

```diff
@@ -233,22 +233,8 @@ private Snapshot loadLocalSnapshot() throws SnapshotException, SpentAddressesExc
         return null;
     }
 
-    /**
-     * <p>
-     * Loads the builtin snapshot (last global snapshot) that is embedded in the jar (if a different path is provided it
-     * can also load from the disk).
-     * </p>
-     * <p>
-     * We first verify the integrity of the snapshot files by checking the signature of the files and then construct
-     * a {@link Snapshot} from the retrieved information.
-     * </p>
-     * <p>
-     * We add the NULL_HASH as the only solid entry point and an empty list of seen milestones.
-     * </p>
-     * @return the builtin snapshot (last global snapshot) that is embedded in the jar
-     * @throws SnapshotException if anything goes wrong while loading the builtin {@link Snapshot}
-     */
-    private Snapshot loadBuiltInSnapshot() throws SnapshotException {
+    @Override
+    public Snapshot loadBuiltInSnapshot() throws SnapshotException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'loadBuiltInSnapshot()' has a cyclomatic complexity of 12.](https://app.codacy.com/app/GalRogozinski/iri/pullRequest?prid=4541389)